src/core/qgismobileapp.h

```diff
@@ -117,6 +118,14 @@ class QgisMobileapp : public QQmlApplicationEngine
       return singletonClass;
     }
 
+    static QObject *geometryEditorsSingletonProvider( QQmlEngine *engine, QJSEngine *scriptEngine )
+    {
+      Q_UNUSED( engine )
```

Comment:
Code to cpp?

Reply:
What do you mean?

Reply:
Move the definition from the header to the cpp file?